src/routes/api/locationsRoute.js

```diff
@@ -138,7 +134,7 @@ router.get('/', permit(['view locations']), getLocations);
  *
  *
  */
-router.get('/:id', permit(['view locations']), getOneLocation);
+router.get('/:id', getOneLocation);
```

Comment:
the same here , please keep plotecting this route

Reply:
@salviosage fixed ✅